main/src/cgeo/geocaching/storage/DataStore.java

```diff
@@ -83,17 +84,14 @@
 import org.apache.commons.collections4.MapUtils;
 import org.apache.commons.lang3.ArrayUtils;
 import org.apache.commons.lang3.StringUtils;
+import org.json.JSONException;
 
 public class DataStore {
 
     public static final String DB_FILE_NAME = "data";
     public static final String DB_FILE_NAME_BACKUP = "cgeo.sqlite";
     public static final String DB_FILE_CORRUPTED_EXTENSION = ".corrupted";
 
-    private DataStore() {
```

Comment:
The modification was made as my changes to 'WAYPOINT_COLUMNS' (line 163) and 'dbVersion' (line 173) were raising Codacy warnings because they were coming after the constructor.